Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -0,0 +1,110 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  SUPPORTED_REPOS = [
+    %w[brew core cask],
+    OFFICIAL_CMD_TAPS.keys.map { |t| t.delete_prefix("homebrew/") },
+    OFFICIAL_CASK_TAPS,
+  ].flatten.freeze
+
+  sig { returns(CLI::Parser) }
+  def contributions_args
+    Homebrew::CLI::Parser.new do
+      usage_banner "`contributions` <email|name> <repo1,repo2|all>"
+      description <<~EOS
+        Contributions to Homebrew repos for a user.
+
+        The first argument is a name (e.g. "BrewTestBot") or an email address (e.g. "brewtestbot@brew.sh").
+
+        The second argument is a comma-separated list of repos to search.
+        Specify <all> to search all repositories.
+        Supported repositories: #{SUPPORTED_REPOS.join(", ")}.
+      EOS
+
+      flag "--from=",
+           description: "Date (ISO-8601 format) to start searching contributions."
+
+      flag "--to=",
+           description: "Date (ISO-8601 format) to stop searching contributions."
+
+      named_args [:email, :name, :repositories], min: 2, max: 2
```

Comment:
Seems a bit weird to give these arguments names that you never use.

Reply:
These are useful if we're automatically generating the usage string, but since we're overriding and don't really want any autocomplete, I think we can just remove them:

```suggestion
      named_args min: 2, max: 2
```